### PR TITLE
data:kushvahasumit

### PR DESCRIPTION
**Pull request by kushvahasumit**
### fussed for my  LinkFree page 🥳

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3959"></a>

### CODE REVIEW

**Review by yashdev9274 — CHANGES_REQUESTED** [review]
Hey @kushvahasumit the JSON format is not correct, you can take a reference from other profiles and update the changes as required

File: data/kushvahasumit.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+},
+{
+  "name" : "subscribe my YouTube",
+  "url" : "https://youtube.com/@samcode130",
+  "icon" : "FaYoutube"
+},
+{
+  "name" : "my Hashnode blog",
+  "url" : "https://hashnode.com/@kushvaha687",
+  "icon" : "FaHashnode"
```
Comment (Njong392) [inline data/kushvahasumit.json:31]: This is not the right icon for hashnode. Try,
```suggestion
  "icon" : "SiHashnode"
```

File: data/kushvahasumit.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  "url" : "https://twitter.com/ur_samy007",
+  "icon" : "FaTwitter"
+},
+{
+  "name" : "follow me on GitHub",
+  "url" : "https://github.com/kushvahasumit",
+  "icon" : "FaGitHub"
+},
+{
+  "name" : "subscribe my YouTube",
```
Comment (Njong392) [inline data/kushvahasumit.json:24]: Could you have possibly skipped a word here?
```suggestion
  "name" : "subscribe to my YouTube",
```

File: data/kushvahasumit.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
+{
+  "name" : "sumit",
+  "displayStatusPublic" : true,
+  "type" : "personal",
+  "bio" : "Undergraduate || new for open source || enthusiast about tech world || web dev",
+  "avatar" : "https://github.com/kushvahasumit.png",
+  "links" : [
+  {
+  "name" : "follow me on Linkedin",
+  "url" : "https://www.linkedin.com/in/sumit-kumar-4a1344254",
```
Comment (Njong392) [inline data/kushvahasumit.json:10]: Appears that the LinkedIn link you have here goes to a 404 page.

**Review by Njong392 — CHANGES_REQUESTED** [review]
Thank you very much for contributing. Please check out the changes I have suggested.

File: data/kushvahasumit.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+{
+  "name" : "my Hashnode blog",
+  "url" : "https://hashnode.com/@kushvaha687",
+  "icon" : "FaHashnode"
+},
+{
+  "name" : "follow me on Instagram",
+  "url" : "https://instagram.com/the_fellow_person?igshid=Mzc0YWU1OWY=",
+  "icon" : "FaInstagram"
+},
```
Comment (ChinmayMhatre) [inline data/kushvahasumit.json:37]:
```suggestion
}
]
}
```

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
I few missing brackets please commit these changes

File: data/kushvahasumit.json (hunk 4 above)
Comment (Njong392) [inline data/kushvahasumit.json:37]: You seem to have ended the JSON wrongly. This won't pass the checks, unfortunately.
```suggestion
  }
 ]
}
```

**Review by Njong392 — CHANGES_REQUESTED** [review]

File: data/kushvahasumit.json (hunk 3 above)
Comment (kushvahasumit) [reply data/kushvahasumit.json:10]: sorry for all issues, i fixed the linkedin link but i didn't have any idea how to change my code please guide me.

Comment (Njong392) [reply data/kushvahasumit.json:10]: Don't say sorry for learning :+1: 
- Go back to your fork (Sync your fork with this upstream repo as that's always good)
- Go the required `json` file and you can directly edit in your Github UI.
- Make all the necessary corrections, and at the bottom you'll be required to to commit those new changes.
- This PR will get updated with your corrections.

If there is any issue, feel free to ask.

Comment (Njong392) [reply data/kushvahasumit.json:10]: Something else to note here,

If you've made the changes on your local computer, and not in the Github UI, all you have to do is commit and push. The PR will still automatically get updated.

Comment (kushvahasumit) [reply data/kushvahasumit.json:10]: I delete this , and i make a new pr.

**Pull request closed without merging**